kyo-core/shared/src/test/scala/kyoTest/loopsTest.scala

```diff
@@ -0,0 +1,293 @@
+package kyoTest
+
+import kyo.*
+
+class loopsTest extends KyoTest:
```

Comment:
Perhaps it's worth adding a test with a `List` accumulator - assuming we want users referencing tests. It seems more inline with how I might use this in practical application.